docs/user/aci_commands.md

```diff
@@ -0,0 +1,40 @@
+# Cisco ACI Chat Commands
+
+## Screenshots
```

Comment:
I would move the screenshot section to the end. They are a bit easier to understand after reading through the command reference.

Reply:
Moved